analyses/copy_number_consensus_call/Snakefile

```diff
@@ -33,11 +33,11 @@ rule freec_filter:
         ## The first awk looks at column 6 to filter out for loss/gain. Then it prints out 6 of the 7 columns above
         ## The pipe into the second awk filters the CNV length, freec pval, and add in the CNV type
         ## The last pipe is to sort first digit of chromosome number numerically
-        """awk '$6~/loss/ {{print "chr"$2,$3,$4,($4-$3 + 1),$5,$9}}' {input.events} """
+        """awk '$6~/loss/ {{print "chr"$2,$3,$4,($4-$3 + 1),$5,$9,"NA"}}' {input.events} """
         """ | awk '{{if ($4 > {params.SIZE_CUTOFF} && $6 < {params.FREEC_PVAL}){{print $0,"DEL"}}}}' """
```

Comment:
For clarity, perhaps we should move the "DEL" addition here (and "DUP", similarly) to the first awk statement. So this could become:
```
"""awk '$6~/loss/ {{print "chr"$2,$3,$4,($4-$3 + 1),$5,$9,"NA","DEL"}}' {input.events} """
""" | awk '{{if ($4 > {params.SIZE_CUTOFF} && $6 < {params.FREEC_PVAL}){{print}}}}' """
```
The same simplification could be applied to the CNVkit awk statements.